<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>concat</class>
 <widget class="QWidget" name="concat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频合并/分割/调整大小</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>611</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>路径</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_Split">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>分割视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Refresh">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Merge">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>合并视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Resize">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>调整大小</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
